content/docs/guides/converting-md-to-mdx.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Converting Gatsby Markdown to MDX for TinaCMS
```

Comment:
```suggestion
title: Markdown to MDX migration guide
```
This guidance could work for any markdown, not just Gatsby. Also, the title could be confusing given that there is a Gatsby migration guide already.